core/src/Kokkos_Macros.hpp

```diff
@@ -183,8 +183,8 @@
 // Intel compiler macros
 
 #if defined(KOKKOS_COMPILER_INTEL)
-// FIXME_SYCL
-#if !defined(KOKKOS_ENABLE_SYCL)
+// FIXME_ICPX
+#if !defined(__INTEL_LLVM_COMPILER)
```

Comment:
What the deal with that change?

Reply:
The compiler is complaining about not knowing the corresponding pragmas. I initially observed that when using `SYCL` but it also seems to appear without it.
I would discuss the existence of these macros anyway. IMHO, they only make sense if they actually expanded to a pragma.